### PR TITLE
fix backend test errors

### PR DESCRIPTION
**Pull request by yhedholm**
I made the backed errors run without errors.

### CODE REVIEW

**Comment by yhedholm** [comment]
Sorry! I meant "I made the backend error tests run without errors".

File: backend/src/main/java/com/google/rolecall/models/User.java [hunk 1]
```diff
@@ -359,14 +359,14 @@ public Builder setEmail(String email) {
...
+      if(notificationEmail != null) {
         this.notificationEmail = notificationEmail.toLowerCase();
       }
       return this;
     }
   
     public Builder setPictureFile(String pictureFile) {
-      if(email != null) {
+      if(pictureFile != null) {
         this.pictureFile = pictureFile.toLowerCase();
```
Comment (RichardYang2016) [inline backend/src/main/java/com/google/rolecall/models/User.java:370]: I believe this should be made lowercase. in URL case is sensitive.

**Review by RichardYang2016 — COMMENTED** [review]
a potential bug pls check comment. thanks

File: backend/src/main/java/com/google/rolecall/models/User.java (hunk 1 above)
Comment (yhedholm) [reply backend/src/main/java/com/google/rolecall/models/User.java:370]: Done.

Comment (RichardYang2016) [reply backend/src/main/java/com/google/rolecall/models/User.java:370]: sorry i had a typo. I was saying this should NOT be made lowercase.

basically you can change to 
  this.pictureFile = pictureFile.
and remove the toLowerCase() call 

Thanks
Richard

**Comment by RichardYang2016** [comment]
sorry i had a typo. I was saying this should NOT be made lowercase.

basically you can change to
this.pictureFile = pictureFile.
and remove the toLowerCase() call

Thanks
Richard

**Review by RichardYang2016 — APPROVED** [review]

**Pull request merged**